qpc/report/tests_report_merge_status.py

```diff
@@ -67,8 +67,8 @@ def test_job_valid_id(self):
             args = Namespace(job_id='1')
             with redirect_stdout(report_out):
                 nac.main(args)
-                self.assertIn(messages.JOB_ID_STATUS % ('1',
-                                                        'complete'),
+                self.assertIn(messages.MERGE_JOB_ID_STATUS % ('1',
```

Comment:
same question here as above

Reply:
The reason for this is that there are multiple lines of output.  I would argue that this test is fine the way it is but I did update it.